### ISSUE TITLE
Loss goes to nan when uses res101  

### ISSUE THREAD

**Issue by John1231983** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#12]
I am training your code with res101 in cityscape dataset (for deeplab pre-trained it worked well). I set up the `main.py` as 
```
flags.DEFINE_float('momentum', 0.9, 'momentum')
	flags.DEFINE_string('encoder_name', 'res101', 'name of pre-trained model, res101, res50 or deeplab')
	flags.DEFINE_string('pretrain_file', './reference model/resnet_v1_101.ckpt', 'pre-trained model filename corresponding to encoder_name')
	flags.DEFINE_string('data_list', './dataset_cityscapes/train_fine.txt', 'training data list filename')
...
	flags.DEFINE_integer('input_height', 713, 'input image height')
	flags.DEFINE_integer('input_width', 713, 'input image width')
	flags.DEFINE_integer('num_classes', 19, 'number of classes')
```

After some iterations, the loss goes to nan. I am using python3 and tensorflow 1.3. Did you meet same problem as me? How could I fix it? Thanks

This is loss log

```
step 0 	 loss = 6.337, (12.625 sec/step)
step 1 	 loss = 6.133, (1.820 sec/step)
step 2 	 loss = 2.675, (1.625 sec/step)
step 3 	 loss = 6.042, (1.630 sec/step)
step 4 	 loss = 15.278, (1.545 sec/step)
step 5 	 loss = 12.153, (1.532 sec/step)
step 6 	 loss = 52.724, (1.100 sec/step)
step 7 	 loss = 940.443, (1.041 sec/step)
step 8 	 loss = 5393914151199927058379743980628738048.000, (1.052 sec/step)
step 9 	 loss = nan, (1.093 sec/step)
step 10 	 loss = nan, (1.445 sec/step)
step 11 	 loss = nan, (1.025 sec/step)

```

**Comment by John1231983** [comment]
Sorry. I may be used too larger learning rate. I closed the issue
